package.json

```diff
@@ -2,10 +2,46 @@
   "name": "rpc-websockets",
   "version": "8.0.1",
   "description": "JSON-RPC 2.0 implementation over WebSockets for Node.js",
-  "main": "./dist/index.cjs",
-  "types": "./dist/index.d.cts",
+  "browser": "dist/index.browser.mjs",
+  "main": "dist/index.cjs",
+  "module": "dist/index.mjs",
+  "types": "dist/index.d.ts",
+  "targets": {
+    "browser": {
+      "isLibrary": true,
+      "outputFormat": "esmodule",
+      "source": "src/index.browser.ts",
+      "sourceMap": true
+    },
+    "main": {
+      "isLibrary": true,
+      "outputFormat": "commonjs",
+      "source": "src/index.ts",
+      "sourceMap": true
+    },
+    "module": {
+      "context": "node",
+      "isLibrary": true,
+      "outputFormat": "esmodule",
+      "source": "src/index.ts",
+      "sourceMap": true
+    },
+    "browser-bundle": {
+      "context": "browser",
+      "distDir": "dist/",
+      "outputFormat": "global",
+      "source": "src/index.browser-bundle.ts"
+    },
+    "types": {
+      "source": "src/index.ts"
+    }
+  },
```

Comment:
This is the complete Parcel configuration. When you run `parcel build` it reads _this_ data to know how to behave.